content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,250 @@ the product page, you can see the following sections:
-in a namespace are accessible by services from another namespace.
-
-In our Bookinfo sample, the `productpage`, `reviews`, `details`, `ratings` services are deployed in the `default` namespace.
-The Istio components like `istio-ingressgateway` service are deployed in the `istio-system` namespace. We can define a policy that
-any service in the `default` namespace that has the `app` label set to one of the values of
-`productpage`, `details`, `reviews`, or `ratings`
-is accessible by services in the same namespace (i.e., `default`) and services in the `istio-system` namespace.
-
-Run the following command to create a namespace-level access control policy:
-
-{{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_apply.sh" >}}
+## Enforce mesh-level access control
 
-Once applied, the policy has the following effects:
+Using Istio, you can easily setup mesh-level access control for all workloads in your mesh.
 
-*   Creates a `ServiceRole` `service-viewer` which allows read access to any service in the `default` namespace that has
-the `app` label
-set to one of the values `productpage`, `details`, `reviews`, or `ratings`. Note that there is a
-constraint specifying that
-the services must have one of the listed `app` labels.
+Run the following command to create a mesh-level access control policy `deny-all` policy
+in the root namespace, A policy in the root namespace will select all workloads in the mesh.
 
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_service_viewer.yaml" >}}
+The root namespace is configurable in the [`MeshConfig`](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig)
+and has the default value of `istio-system`. If you have changed it to a different value from the
+default `istio-system`, Please update the value accordingly in the following examples.
 
-*   Creates a `ServiceRoleBinding` that assigns the `service-viewer` role to all services in the `istio-system` and `default` namespaces.
-
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enforcing_namespace_level_access_control_bind_service_viewer.yaml" >}}
-
-You can expect to see output similar to the following:
-
-{{< text plain >}}
-servicerole "service-viewer" created
-servicerolebinding "bind-service-viewer" created
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+  namespace: istio-system
+spec:
+  {}
+EOF
 {{< /text >}}
 
-Now if you point your browser at Bookinfo's `productpage` (`http://$GATEWAY_URL/productpage`). You should see the "Bookinfo Sample" page,
-with the "Book Details" section in the lower left part and the "Book Reviews" section in the lower right part.
+The `{}` in the policy means `deny-all`, you can also set other mesh-level policy for your own needs.
```

Comment:
This phrase may confuse users that this is a deny rule. In fact, `{}` is just a general yaml syntax for empty. 
Suggest to reword to smt like:

```
This policy has no rule so it cannot grant permission for any traffic. In other words, it denies all.
```